### PR TITLE
Add missing variable declaration MST_NAMESPACE

### PR DESCRIPTION
**Pull request by kakkoyun**
Signed-off-by: Kemal Akkoyun <kakkoyun@gmail.com>

1. Add missing declaration
2. Sync metrics allow list

### CODE REVIEW

File: services/observatorium-template.jsonnet [hunk 1]
```diff
@@ -15,9 +15,9 @@ local obs = import 'observatorium.libsonnet';
     ],
   parameters: [
     { name: 'NAMESPACE', value: 'observatorium' },
+    { name: 'MST_NAMESPACE', value: 'observatorium-mst-production' },
```
Comment (squat) [inline services/observatorium-template.jsonnet:18]: this should generate a diff too, right?

**Review by squat — COMMENTED** [review]
Thanks for tracking this down!

File: services/observatorium-template.jsonnet (hunk 1 above)
Comment (kakkoyun) [reply services/observatorium-template.jsonnet:18]: Yes, I'll add those soon

**Review by squat — APPROVED** [review]
great :penguin:

**Pull request merged**